### PR TITLE
Feat/dialog notes word count

### PR DESCRIPTION
**Pull request by SNujiy**
Add word count for note and a max word restrictions.

Should smileys such as ":D" count as words? - As Boyum suggested, smileys will be counted as a word.

### CODE REVIEW

File: src/components/Dialog-Window/Notes/DialogNote.tsx [hunk 1]
```diff
@@ -34,10 +35,23 @@ export const Note: React.FC<NoteProps> = ({ maxLength, id }) => {
...
 
+  const countWords = (): void => {
+    let count = 0;
+    const words = note.split(" ");
+    words.forEach(s => {
+      const trim = s.trim();
+      if (trim.length > 0) {
+        count += 1;
+      }
+    });
```
Comment (boyum) [inline src/components/Dialog-Window/Notes/DialogNote.tsx:46]: Can be functionalized, but I'm not sure if it's any cleaner

```suggestion
const count = note.split(" ")
  .map(s => s.trim())
  .filter(Boolean)
  .length;
```

**Comment by boyum** [comment]
I think ":D" should be counted as a word, just because I don't think it's a good idea to check every word against a list of "non-word-words" (which might be different in every language).

File: src/components/Dialog-Window/Notes/DialogNote.tsx (hunk 1 above)
Comment (SNujiy) [reply src/components/Dialog-Window/Notes/DialogNote.tsx:58]: I think it is cleaner, will update!

**Review by boyum — APPROVED** [review]
NICE

**Pull request merged**